doc/FAQ.rst

```diff
@@ -4,6 +4,117 @@ FAQ
+  curve = curve.redim.range(x_col=(0, None), z_col=(None, 10))
+
+If your column names have spaces, you may predefine a dictionary
+and unpack it in obj.redim.range(). This same method is applicable to
+adjust the range of a color bar.
+
+.. code:: python
+
+  bounds = {'x col': (0, None), 'z col': (None, 10)}
+  curve = hv.Curve(df, 'x col', ['y col', 'z col']).options(color_index='z col')
+  curve = curve.redim.range(**bounds)
+
+**Q: How do I export a figure?**
+
+**A:** Create a renderer object by passing a backend (matplotlib / bokeh)
+and pass the object and name of file without any suffix into the .save method.
+
+.. code:: python
+
+  backend = 'bokeh'
+  renderer = hv.renderer(backend)
+  renderer.save(obj, 'name_of_file')
+
+Just note that if have used Jupyter magic (%%opts) to customize your plot
+and you try to export it, the customizations will not be retained in the export.
+
+.. code:: python
+
+  %%opts Curve [width=1000]
+  curve = hv.Curve([1, 2, 3])
+  hv.renderer('bokeh').save(curve, 'example_curve')  # width will not be saved
+
+You'll have to pass the keyword arguments into .options().
+
+.. code:: python
+
+  curve = hv.Curve([1, 2, 3]).options(width=1000)
+  hv.renderer('bokeh').save(curve, 'example_curve')
+
+**Q: How do I provide axes labels?**
+
+**A:** Pass a tuple containing the column name and label.
+
+This will relabel 'x_col' to 'X Label'
+.. code:: python
+  curve = hv.Curve(df, ('x_col', 'X Label'), 'y_col')
+
+You may also label after the fact by passing an unpacked dictionary
+to .redim.label()
+.. code:: python
+  curve = hv.Curve(df, 'x_col', 'y_col')
+  curve = curve.redim.label(x_col='X Label', **{'y_col': 'Label for Y'})
+
+**Q: How can I access all the options that aren't exposed in HoloViews,
+but are available in the backend?**
+
+**A:** There are two approaches you can take.
+
+The first is converting HoloViews objects as bokeh/matplotlib figures,
+and then continuing to work on those figures natively in the selected backend.
+
+.. code:: python
+  backend = 'matplotlib'
+  hv_obj = hv.Curve(df, 'x_col', 'y_col')
+  fig = hv.renderer(backend).get_plot(hv_obj).state
+  # this is just a demonstration; you can directly relabel in HoloViews
+  fig.axes[0].set_xlabel('X Label')
+
+The second is through finalize_hooks (bokeh) / final_hooks (matplotlib)
+which helps retain a HoloViews object.
+
+.. code:: python
+   def relabel(plot, element):
+       # this is for demonstration purposes
+       # use the .redim.label() method instead!
+       fig = plot.state
+       fig.axes[0].set_xlabel('X Label')
+
+  backend = 'matplotlib'
+  hv_obj = hv.Curve(df, 'x_col', 'y_col')
+  hv_obj = hv_obj.options(final_hooks=[relabel])
+
+**Q: The default figure size is so tiny! How do I enlarge it?**
+
+**A:** Depending on the selected backend...
+
+.. code:: python
+    # for matplotlib:
+    hv_obj = hv_obj.options(fig_size=500)
+
+    # for bokeh:
+    hv_obj = hv_obj.options(width=1000, height=500)
+
```

Comment:
I personally never thought of it like that; I just assumed different keyword arguments with each backend, but I did always wonder why HoloViews kwarg for matplotlib's backend is `fig_size` instead of matplotlib's `figsize`, and also why in matplotlib you could pass a tuple like `plt.figure(figsize=(8, 6))` but here, it's like just a percentage of the original size?